early-attack-detector-py/src/agent.py

```diff
@@ -275,12 +285,12 @@ def detect_malicious_contract(
             function_signatures = get_function_signatures(w3, opcodes)
             logger.info(f"{created_contract_address}: score={model_score}")
 
-            if model_score is None or model_score < MODEL_INFO_THRESHOLD:
+            if model_score is None or model_score < MODEL_INFO_THRESHOLD and not from_eoa_in_tp_list:
```

Comment:
```suggestion
            if (model_score is None or model_score < MODEL_INFO_THRESHOLD) and not from_eoa_in_tp_list:
```

I think parentheses are needed here